go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.3.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
-	github.com/gruntwork-io/terratest v0.38.5
+	github.com/gruntwork-io/terratest v0.38.6
```

Comment:
Unless the upgrade is needed for this PR, I'd recommend pulling it out to reduce the scope and to make the update more easily cherry-pickable.

Reply:
Something is up with my dev env and `make generate` which causes this. I usually catch it and revert prior to PR. I'll look into root cause soon.